tools/plugins/com.liferay.ide.portlet.core/src/com/liferay/ide/portlet/core/operation/NewPortletClassDataModelProvider.java

```diff
@@ -1024,7 +1024,7 @@ else if( CREATE_RESOURCE_BUNDLE_FILE_PATH.equals( propertyName ) )
             }
             else
             {
-                return PortletCore.createWarningStatus( Msgs.resourceBundleFilePathEndWithProperties );
```

Comment:
This is only for the buildLang command.  What if the developer doesn't want to use our "buildLang" command but wants to manually put in their own language files?  The portlet can be configured with any resource bundle.  So there are many cases where the user can name this file some other name.  The only thing that doens't work is the "buildLang" command.  But the resource bundle will still work in the portal.